src/Files.App/Views/Shells/ModernShellPage.xaml.cs

```diff
@@ -194,7 +189,7 @@ private async void KeyboardAccelerator_Invoked(KeyboardAccelerator sender, Keybo
 			{
 				// Ctrl + V, Paste
 				case (true, false, false, true, VirtualKey.V):
-					if (!ToolbarViewModel.IsEditModeEnabled && !ContentPage.IsRenamingItem && !InstanceViewModel.IsPageTypeSearchResults && !ToolbarViewModel.SearchHasFocus)
+					if (!ContentPage.IsRenamingItem && !InstanceViewModel.IsPageTypeSearchResults)
```

Comment:
This seems to be causing a regression, do you have any suggestions? Do we even need this anymore with the actions?